src/main/java/seedu/financialplanner/goal/Goal.java

```diff
@@ -12,15 +12,17 @@ public Goal(String label, int amount) {
 
     public String toString() {
         String status = isDone ? "Done" : "Not Done";
-        return "Goal: " + this.label + " | " + this.amount + " | " + status;
+        return "Goal " + System.lineSeparator()+ "   Label: " + label + System.lineSeparator() + "   Amount: " +
```

Comment:
Should there be a ":" after Goal? Same thing for reminder

Reply:
This part I follow the expense, which is without a ":".